validation/src/test/scala/hmda/validation/engine/lar/quality/LarQualityEngineSpec.scala

```diff
@@ -0,0 +1,26 @@
+package hmda.validation.engine.lar.quality
+
+import java.io.File
+
+import hmda.parser.fi.lar.{ LarCsvParser, LarGenerators }
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+import scala.io.Source
+
+class LarQualityEngineSpec
+    extends PropSpec
+    with PropertyChecks
+    with MustMatchers
+    with LarGenerators
+    with LarQualityEngine {
+
+  property("A LAR must pass quality checks") {
+    val lines = Source.fromFile(new File("parser/src/test/resources/txt/FirstTestBankData_clean_407_2017.txt")).getLines()
+    val lars = lines.drop(1).map(l => LarCsvParser(l))
+
+    lars.foreach { lar =>
+      checkQuality(lar).isSuccess mustBe true
+    }
```

Comment:
Just a warning here, these files were checked for syntactical and validity edits only. They may fail some quality and macro edits.